app/views/migrations/_most_recent_migration.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<h2 class="govuk-heading-l">Latest migration</h2>
+
+<p class="govuk-body">
+  The latest migration was completed <strong><%= time_ago_in_words(@most_recent_migration.completed_at) %></strong> ago.<br>
+  The migration took <strong><%= migration_result_duration_in_words(@most_recent_migration) %></strong> to complete.
+</p>
+
+<div class="govuk-grid-row">
+  <% %w[users applications].each do |key| %>
+    <div class="govuk-grid-column-one-half">
+      <%= tag.div(class: "govuk-summary-card #{key}") do %>
+        <div class="govuk-summary-card__title-wrapper">
+          <h2 class="govuk-summary-card__title"><%= key.pluralize.titleize %> Reconciliation</h2>
+          <ul class="govuk-summary-card__actions">
+            <li class="govuk-summary-card__action">
+              <%= link_to("Orphan report", download_orphan_report_migrations_path(@most_recent_migration.id, key)) %>
+            </li>
+          </ul>
+        </div>
+        <div class="govuk-summary-card__content">
+          <dl class="govuk-summary-list">
+            <% migration_result_attributes(@most_recent_migration, key).each do |attribute| %>
+              <%= migration_result_summary_row(@most_recent_migration, key, attribute) %>
+            <% end %>
+          </dl>
+        </div>
+      <% end %>
```

Comment:
Doesn't matter too much for this but [these are fully supported in govuk-components](https://govuk-components.netlify.app/components/summary-list/#summary-cards).

Reply:
Something along the lines of:

```erb
<%=
  govuk_summary_card(title: "#{key.pluralize.titleize} Reconciliation") do |card|
    card.with_action do
      govuk_link_to("Orphan report", download_orphan_report_migrations_path(@most_recent_migration.id, key))
    end
    card.with_summary_list(rows: migration_result_summary_rows)
  end
%>
```

Reply:
Thanks, I fell back to plain HTML here as I'm misusing the component a bit; the `govuk-components` seems to require the action to be a link (which makes sense and aligns with the component guidelines). It means I can't use the helpers though; unless I'm missing a way of getting a tag into the row actions, but from my understanding it expects a hash with the attributes for the underlying slot, something like:

```
[{
  key: { text: "Row title" },
  value: { text: "Row value" },
  actions: [<these have to be links>],
}]
```